question-servers/freeformPythonLib/prairielearn.py

```diff
@@ -41,6 +41,14 @@ def _get_attrib(element, name, *args):
             raise Exception("Attribute \"%s\" missing and no default is available" % name)
     return (element.attrib[name], False)
 
+def is_attrib(element, name):
```

Comment:
Maybe `has_attrib()` might be more in keeping with python built-ins like `hasattr()`?